File: CStoreManagement/app/src/main/java/com/c_store/zhiyazhang/cstoremanagement/view/interfaceview/ContractTypeView.java
package com.c_store.zhiyazhang.cstoremanagement.view.interfaceview;

import android.view.View;

import com.c_store.zhiyazhang.cstoremanagement.bean.ContractTypeBean;
import com.c_store.zhiyazhang.cstoremanagement.bean.UserBean;
import com.c_store.zhiyazhang.cstoremanagement.presenter.contract.ContractTypeAdapter;

/**
 * Created by zhiya.zhang
 * on 2017/5/9 16:41.
 */

public interface ContractTypeView {
    void toShortClick(View view, ContractTypeBean ctb);

    void toLongClick(View view);

    void showLoading();

    void hideLoading();

    void showView(ContractTypeAdapter adapter);

    void showFailedError(String errorMessage);
}

File: KotlinCStoreManagement/app/src/main/java/com/cstore/zhiyazhang/cstoremanagement/view/interface_view/ContractTypeView.kt
package com.cstore.zhiyazhang.cstoremanagement.view.interface_view

import com.cstore.zhiyazhang.cstoremanagement.bean.ContractTypeBean
import com.cstore.zhiyazhang.cstoremanagement.presenter.contract.ContractTypeAdapter

/**
 * Created by zhiya.zhang
 * on 2017/6/12 15:08.
 */
interface ContractTypeView {
    fun toActivity(ctb: ContractTypeBean)
    fun showLoading()
    fun hideLoading()
    fun showView(adapter: ContractTypeAdapter)
    fun showFailedError(errorMessage: String)
}